### ISSUE TITLE
Bizarre geographic patterns in this dataset

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5509]
**Bizarre geographic patterns in this dataset**

Zooming in on the map there is a striking pattern of parallel lines at a 45° angle from bottom left to top right over Brazil. I first noticed these on the map on the home page, they are more noticeable here (you have to zoom in a bit to see them).

There is clearly a massive geographic artefact affecting this dataset. Perhaps a grid has been encoded as a corner point?

-----

Github user: @rdmpage
User: [See in registry](https://www.gbif.org/api/feedback/user/bb81e4084a3f9130963b6d112465864a:a078369ff022aac013b87471e9bf579d1042f1c8b39dc33cba9d5c01f2afd393997ce9eb4e133ec3bc2e68ad818ebb10c4bd10a49d2beac3e82e97f1bc54997d4c3b80ed19032b9834915b2d4058623970c0c3f1166776ce697fedcb8fc80b360657cff4e18c3f4fe3cffc1fe33324ec38d03e27269a123f934c680cc2c52a1b) - [Send email](https://www.gbif.org/api/feedback/user/mailto/bb81e4084a3f9130963b6d112465864a:a078369ff022aac013b87471e9bf579d1042f1c8b39dc33cba9d5c01f2afd393997ce9eb4e133ec3bc2e68ad818ebb10c4bd10a49d2beac3e82e97f1bc54997d4c3b80ed19032b9834915b2d4058623970c0c3f1166776ce697fedcb8fc80b360657cff4e18c3f4fe3cffc1fe33324ec38d03e27269a123f934c680cc2c52a1b)
System: Safari 17.4.1 / Mac OS X 10.15.7
Referer: https://www.gbif.org/dataset/2e684fd6-09e2-4661-910a-0a188b610c04
Window size: width 1151 - height 892
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-10-11T16:01:28.971Z',to:'2024-10-11T16:07:28.971Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-10-11T16:01:28.971Z',to:'2024-10-11T16:07:28.971Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by rdmpage** [comment]
I've also made a comment on the GBIF community site https://discourse.gbif.org/t/bizarre-geographic-patterns-in-brazil/5408. Below are maps from the GBIF home page and the dataset home page.

![Image](https://github.com/user-attachments/assets/73cca856-af15-4edf-b4c5-a649278cd23b)
![Image](https://github.com/user-attachments/assets/f490619f-25a9-46e0-ae73-f74358fd9294)

**Comment by sformel-usgs** [comment]
Yeah, it’s pretty strange. I’m guessing someone made a georeferencing decision but didn’t document it. This data point gives a sense of how ‘off’ the coordinates might be.

- This occurrence: [Occurrence Detail 4928891896](https://www.gbif.org/occurrence/4928891896)
- has a locality of "behind the Byzantine cemetery in Mucugê, Bahia, Brasil: https://maps.app.goo.gl/t2cxezruVLqg99Ny5
- That google maps pin is ~40km away from the DwC coordinates.

Anyone have any time to reach out to the dataset POCs? I'm too swamped to follow up on this right now.

**Comment by CecSve** [comment]
@ClaraBaringoFonseca would you be able to follow up with this Brazilian publisher perhaps?
